OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -56,13 +56,14 @@ public sealed class HackyAIInfo : IBotInfo, ITraitInfo
 		[Desc("Minimum excess power the AI should try to maintain.")]
 		public readonly int MinimumExcessPower = 0;
 
-		[Desc("Delay (in ticks) between structure production checks when there is no active production.",
-			"A StructureProductionRandomBonusDelay is added to this.")]
+		[Desc("Additional delay (in ticks) between structure production checks when there is no active production.",
+			"StructureProductionRandomBonusDelay is added to this.")]
 		public readonly int StructureProductionInactiveDelay = 125;
 
-		[Desc("Delay (in ticks) between structure production checks when actively building things.",
-			"A StructureProductionRandomBonusDelay is added to this.")]
-		public readonly int StructureProductionActiveDelay = 10;
+		[Desc("Additional delay (in ticks) added between structure production checks when actively building things.",
+			"Note: The total delay is gamespeed OrderLatency x 3 + this + StructureProductionRandomBonusDelay.",
+			"A value of at least 2 or 3 is recommended to avoid AI issuing duplicate build orders.")]
```

Comment:
Is this last sentence still true? That should be completely solved by the OrderLatency change.